### PR TITLE
Link out to webapp in examples

### PR DESCRIPTION
**Pull request by rudyardrichter**

### CODE REVIEW

File: docs/source/authoring_flows.rst [hunk 1]
```diff
@@ -396,6 +396,9 @@ from a source to a destination and then deleting the directory from the source.
 The entire directory's contents, including files and subdirectories, will be
 moved to the destination and then removed from the source.
 
+View this flow in the `Globus web app. <https://app.globus.org/flows/9123c20b-61e0-46e8-9469-92c999b6b8f2/definition>`_
+(You may need to log in first.)
```
Comment (jakeglobus) [inline docs/source/authoring_flows.rst:400]: I don't know if this matters, but this may also require providing consent.

**Review by jakeglobus — APPROVED** [review]
I verified the links are correct

File: docs/source/authoring_flows.rst (hunk 1 above)
Comment (rudyardrichter) [reply docs/source/authoring_flows.rst:400]: yeah, that whole procedure is what was meant here, I think you do need to consent if you haven't used flows before. do you think there's a better wording?

Comment (jakeglobus) [reply docs/source/authoring_flows.rst:400]: I wouldn't worry about it too much. Looking at it now, I think the wording is fine.

**Pull request merged**